src/Sulu/Bundle/AdminBundle/Resources/public/scss/modules/sticky-toolbar.scss

```diff
@@ -0,0 +1,31 @@
+.sticky-toolbar.fixed {
+    .list-toolbar-container {
+        width: 100%;
+        padding-right: 100px;
+        position: absolute;
+        background: #f1f1f1;
+        z-index: 98;
+        top: 30px;
+        padding-top: 10px;
+
+        &.fixed {
+            padding-right: 100px;
+            width: $contentFixedWidth;
+        }
+
+        @media only screen and (max-width: 50px + $contentFixedSmallWidth + $sidebarMaxCollapseWidth) {
```

Comment:
You have to show me that after lunch, can't find any position where this rule applies.